### ISSUE TITLE
make code blocks full-width

### ISSUE THREAD

**Issue by theonlygusti** [viosey/hexo-theme-material#172]
**I certify that I have first consulted** (check all with "x")

- [x] [Hexo documentation](https://hexo.io/docs/)
- [x] [Material theme documentation](https://material.viosey.com/)
- [x] [Material theme issues](https://github.com/viosey/hexo-theme-material/issues?utf8=%E2%9C%93&q=is%3Aissue)

**I'm submitting a**  (check one with "x")

- [x] bug report
- [ ] feature request
- [ ] support request

This code

    ```java
    import org.bukkit.JavaPlugin;
    
    public class Plugin extends JavaPlugin {
      // just testing out syntax highlighting
      // thus, this is nonsense code xD
    
      public static void main(String[] args) {
        // TODO: write linker
        return 0;
      }
      
      public Plugin(String name) {
        this.name = name;
      }
    }
    ```

generates this

![http://i.cubeupload.com/ULumjT.png](http://i.cubeupload.com/ULumjT.png)

Really, the output should be the same width as its container. 

(Also, #114)

**Comment by theonlygusti** [comment]
Is there an option to disable line numbers?

**Comment by pidupuis** [comment]
@theonlygusti There is no option yet to disable line numbers. The highlight problem is a known problem, see https://github.com/viosey/hexo-theme-material/issues/114
You should have to wait for a further release to have a good code block. For my own blog, I chose to remove comments from my code when they mess up with the display (since I know this is temporary)

**Comment by theonlygusti** [comment]
@pidupuis oh right, what about the width problem though?

Surely it's not intentional that the block is so awkwardly thin.

**Comment by pidupuis** [comment]
@theonlygusti we'll change this as well

**Comment by neoFelhz** [comment]
@pidupuis @theonlygusti We can  disable line numbers of code block in hexo config files.

**Comment by theonlygusti** [comment]
@neoFelhz  What is it?

**Comment by cubesky** [comment]
Fixed in `1.3.0`
已在 `1.3.0` 修复
